src/core/classes/prism.ts

```diff
@@ -134,3 +135,5 @@ export default class Prism {
 		return tokenize.call(this, text, grammar);
 	}
 }
+
+export type { Prism };
```

Comment:
Wouldn't this suffice?
```suggestion
export { Prism };
```